metals/src/main/scala/scala/meta/internal/metals/MetalsServerConfig.scala

```diff
@@ -55,6 +57,14 @@ final case class MetalsServerConfig(
       "metals.warnings",
       default = true
     ),
+    openFilesOnRenames: Boolean = MetalsServerConfig.binaryOption(
+      "metals.open-files-on-renames",
```

Comment:
Nit: All the other names seem to be singular. Could we do `open-files-on-rename`?

Reply:
Sure, I agree it makes sense - changed